src/wx/wxvbam.cpp

```diff
@@ -722,6 +722,7 @@ MainFrame::MainFrame()
 {
     jpoll = new JoystickPoller();
     this->Connect(wxID_ANY, wxEVT_SHOW, wxShowEventHandler(JoystickPoller::ShowDialog), jpoll, jpoll);
+    AttachJoystickHandler(this);
```

Comment:
@v1993 You had the right idea here, It is confusing that it is not the frame that get the input, but the panel (GameArea and DrawingPanelBase classes). After a game is loaded, we Bind from the DrawingPanel to the GameArea functions and on some of them we call `Skip`, which then go to `filterEvents` on our `MainFrame`.